dockergen/nginx.tmpl

```diff
@@ -71,6 +72,7 @@ server {
 {{ range $url, $containers := groupByMulti $containersByHost "Env.VIRTUAL_URL" "," }}
     location {{ $url }} {
         proxy_pass http://{{ $host }}{{ replace $url "/" "-" -1 }}u/;
+        proxy_redirect https://{{ $host }}{{ replace $url "/" "-" -1 }}u/ /;
```

Comment:
Could it be that the container is trying to redirect to https because of some settings and the proxy_pass only works for http?